libsrc/ffdec_lib/src/com/jpexs/decompiler/flash/abc/avm2/parser/pcode/ASM3Parser.java

```diff
@@ -531,7 +531,8 @@ public static AVM2Code parse(Reader reader, AVM2ConstantPool constants, Trait tr
                     case ParsedSymbol.TYPE_KEYWORD_GETTER:
                     case ParsedSymbol.TYPE_KEYWORD_SETTER:
                         if (!(trait instanceof TraitMethodGetterSetter)) {
-                            throw new AVM2ParseException("Unxpected trait type", lexer.yyline());
+                            continue; // Instead of glitching it all the way down, ignore the error.
```

Comment:
Can you explain me why is the exception a problem? What do you want to do?
Please create an issue in the issue tracker, and attach a sample and specify the steps to reproduce the problem.